test/mocks/DelegationMock.sol

```diff
@@ -89,6 +89,11 @@ contract DelegationMock is IDelegationManager {
         return 0;
     }
 
+    function getOperatorShares(
+        address operator,
+        IStrategy[] memory strategies
+    ) external view returns (uint256[] memory) {}
```

Comment:
I'm gonna guess that this needs to return an array with equal length to `strategies`, and that's why a lot of tests are panicing with arr out of bounds

Reply:
actually, tried this out - i think this just needs to be replaced with the actual method rather than an empty method. tests are passing now

Reply:
https://github.com/Layr-Labs/eigenlayer-middleware/pull/170/commits/3688a02f4a7841f2d3befbdd3821cc3b4d22c602#diff-7501823751a3937eed2beae0fb540d70cf5b4abae705e6c6d7a49ccc4e7942dfR95
Good callout, added